app/components/provider_interface/application_timeline_component.rb

```diff
@@ -53,7 +53,7 @@ def note_events
         application_choice.notes.order('created_at').map do |note|
           Event.new(
             'Note added',
-            note.user.full_name,
+            vendor_api_user_full_name(note.user),
```

Comment:
does this account for provider users adding notes through manage, could use `actor_for` instead?

Reply:
@cwrw Updated, you're right, the method needs to check the user type.

Unfortunately `actor_for` would break the timeline if a support user added a note because we'd end up here: https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/helpers/audit_helper.rb#L3 checking `username` against a note record and not an audit record as expected.

Reply:
@cwrw updated - `actor_for` gets complicated if a SupportUser adds a Note but have attempted to handle this case.